pyanaconda/pwpolicy.py

```diff
@@ -135,14 +135,18 @@ def parse(self, args):
     def dataList(self):
         return self.policyList
 
-    def get_policy(self, name):
+    def get_policy(self, name, default=False):
```

Comment:
Nice idea, we can drop the policy getting code in the `input_checking` module with this. :) But I would suggest more descriptive name for the argument, like for example: `fallback_to_default_policy` so that it can be easily seen from the function signature what it does.

Reply:
I've changed the `default` to `fallback_to_default`. It's shorter and still sounds good to me.